bin/compile

```diff
@@ -1,31 +1,13 @@
-#!/usr/bin/env bash
-# bin/compile <build-dir> <cache-dir>
+#!/usr/bin/env ruby
 
-set -e
+# sync output
+$stdout.sync = true
 
-BUILD_DIR=$1
-CACHE_DIR=$2
+$:.unshift File.expand_path("../../lib", __FILE__)
+require "build_pack"
```

Comment:
I was just following [heroku/heroku-buildpack-ruby](https://github.com/heroku/heroku-buildpack-ruby/blob/master/bin/compile#L6), but their use case might be more complex - I'll switch to `require_relative`.